CHANGES.md

```diff
@@ -17,6 +17,7 @@
 - #1437: List of groupchats in modal doesn't scroll
 - #1457: Wrong tooltip shown for "unbookmark" icon
 - #1479: Allow file upload by drag & drop also in MUCs
+- #1487: New config option `muc_respect_autojoin`
```

Comment:
You can now already set it to the URL that it will be when it's released. Take a look at the URLs of other config options. You generally just have to replace underscores with dashes at the end of the URL.

Reply:
Nice. Changed the PR accordingly.